app/src/main/java/com/arygm/quickfix/ui/home/MessagesScreen.kt

```diff
@@ -244,7 +494,10 @@ fun MessageInput(
 
         // Send button
         IconButton(
-            onClick = onSendMessage,
+            onClick = {
+              Log.e("hhaha", "tseft lmessage")
```

Comment:
You can remove this I think :D

Reply:
Thanks! I was using it for debugging and forgot to remove it.